src/main/webapp/index.html

```diff
@@ -1,5 +1,5 @@
 <!--
-  Copyright (c) 2016 IBM Corp.
+  Copyright (c) 2018, 2022 IBM Corp.
```

Comment:
Was changing `2016` to `2018` for correcting a copy/paste problem?  In other words, was this file actually first created in `2018`?